pkg/auth/rbac.go

```diff
@@ -76,30 +68,49 @@ func (c *policyConfig) validateAction(action string) error {
 	return fmt.Errorf("invalid action %s", action)
 }
 
-func (c *policyConfig) validateEnvs(envs string) error {
+func (c *policyConfig) validateEnvs(envs, action string) error {
 	e := strings.Split(envs, ":")
-	// Invalid format
-	if len(e) > 2 || envs == "" {
+	if len(e) != 2 || envs == "" {
 		return fmt.Errorf("invalid environment %s", envs)
 	}
 	// Validate <ENVIRONMENT_GROUP:ENVIRONMENT>
-	if len(e) == 2 {
-		if !valid.EnvironmentName(e[0]) {
-			return fmt.Errorf("invalid environment group %s", envs)
-		}
-		if !valid.EnvironmentName(e[1]) {
-			return fmt.Errorf("invalid environment %s", envs)
-		}
+	if !valid.EnvironmentName(e[0]) {
```

Comment:
```suggestion
       envName := e[0]
       groupName :=e[1]
	if !valid.EnvironmentName(e[0]) {
```

Reply:
Added local variables to make the code more readable.